moveit_experimental/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ COMPONENTS
   trajectory_msgs
   visualization_msgs
   # The following are needed by jog_arm:
-  moveit_msgs
+  control_msgs
```

Comment:
Is [ros-controls/control_msgs](https://github.com/ros-controls/control_msgs) the `control_msgs` package that is referred to here? That [doesn't appear](https://github.com/ros-controls/control_msgs/tree/fbd5cce46383b17504196848ad659f415aa5a2bd/control_msgs/msg) to have a `JointJog.msg`.

Reply:
It's in a PR - we'll have to make sure it gets merged before this one: https://github.com/ros-controls/control_msgs/pull/26